futaba/cogs/tracker/core.py

```diff
@@ -468,6 +557,17 @@ def build_embed(message):
         else:
             self.members_left.append(member)
 
+        blacklist = self.bot.sql.settings.get_tracking_blacklist(member.guild)
+        if blacklist.is_blocked(member):
+            logger.debug(
+                "Ignoring member %s (%d) leaving guild '%s' (%d) due to the user being blacklisted",
+                member.name,
+                member.id,
+                member.guild.name,
+                member.guild.id,
+            )
+            return
+
```

Comment:
:+1:

Reply:
(Though blacklisting join/leave events is definitely an interesting situation. Wonder if there's a good practical usecase for it)